service-admin/docker/app/php-fpm-logging.conf

```diff
@@ -1 +1,2 @@
 access.format = '{"trace_id":"%{HTTP_X_TRACE_ID}e","time_iso8601":"%{%Y-%m-%dT%H:%M:%S%z}T","time_local":"%{%d/%b/%Y:%H:%M:%S %z}T","remote_addr":"%R","real_ip":"%{HTTP_X_REAL_IP}e","real_forwarded_for":"%{HTTP_X_FORWARDED_FOR}e","real_forwarded_proto":"%{HTTP_X_FORWARDED_PROTO}e","request_time":"%d","request_uri":"%{REQUEST_SCHEME}e://%{HTTP_HOST}e%{REQUEST_URI}e","status":"%s","request_method":"%m","http_referrer":"%{HTTP_REFERER}e","http_user_agent":"%{HTTP_USER_AGENT}e","bytes_sent":"%l","http_host":"%{HTTP_HOST}e","server_name":"%{SERVER_NAME}e","server_port":"%{SERVER_PORT}e"}'
+ping.path = '/health'
```

Comment:
Is this standard php-fpm config?

Reply:
Yes, this is standard PHP-FPM setting. It's returns a configurable response ("pong!" by default) on the path specified when a FastCGI request comes in.

https://www.php.net/manual/en/install.fpm.configuration.php